gallery-so/gallery
# [GAL-4224] Show Beta Tester Modal

### Summary of Changes

Add a beta tester invitation on Gallery Editor

### Demo or Before/After Pics



https://github.com/gallery-so/gallery/assets/4480258/6d14d658-ed48-411c-81ff-9ffd4b0db10a




- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots. You can type `/table` to quickly come up with a before/after table.

### Edge Cases

1. Save the gallery with `ADMIN` roles
2. Save the gallery with no roles
3. Modal design on Table view


### Testing Steps

1. Edit your gallery
2. After you click the save button, it should popup the modal (Assume you're not `ADMIN` or `BETA_TESTER` roles)


### Checklist

Please make sure to review and check all of the following:
